src/components/Accordion.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import Icon from './Icon';
+
+class Accordion extends Component {
+  state = { collapsed: true };
+
+  handleClick = () => {
+    this.setState(state => {
+      return {
+        collapsed: !state.collapsed,
+      };
+    });
+  };
+
+  renderIndicator() {
+    const { counter, count } = this.props;
+    const { collapsed } = this.state;
+
+    if (collapsed) {
+      if (counter) return <div className="counter">{count}</div>;
+      return (
+        <div className="chevrons">
+          <Icon name="chevron-down" />
+        </div>
+      );
+    }
+    return (
+      <div className="chevrons">
+        <Icon name="chevron-up" />
+      </div>
+    );
```

Comment:
```js
if (collapsed && counter) {
  return <div className="counter">{count}</div>;
}
return (
  <div className="chevrons">
    <Icon name={`chevron-${collapsed ? "down" : "up"}`} />
  </div>
);
```

Reply:
Wow.. I had a hunch this could be DRYed up further but just couldn't place it.
I love this..!!